core/core.go

```diff
@@ -94,11 +94,11 @@ type IpfsNode struct {
 	PrivateKey ic.PrivKey // the local node's private Key
 
 	// Services
-	Peerstore  pstore.Peerstore     // storage for other Peer instances
-	Blockstore bstore.GCBlockstore  // the block store (lower level)
-	Blocks     bserv.BlockService   // the block service, get/add blocks.
-	DAG        merkledag.DAGService // the merkle dag service, get/add objects.
-	Resolver   *path.Resolver       // the path resolution system
+	Peerstore  pstore.Peerstore       // storage for other Peer instances
+	Blockstore bstore.MultiBlockstore // the block store (lower level)
```

Comment:
@whyrusleeping  I did not design the MultiBlockstore to be able to be swapped out.  (As I see it it, it was designed to allow use of other Blockstores so why would you want to swap it out, but I can understand your viewpoint)   If that is something you want it is likely possible.  I image I can have something done in a few days.